### ISSUE TITLE
typescript error on npm start

### ISSUE THREAD

**Issue by amitport** [microsoft/create-react-app-uifabric#14]
```ts
TypeScript error in .../my-app/node_modules/jest-diff/build/diffLines.d.ts(8,13):
'=' expected.  TS1005

     6 |  */
     7 | import { Diff } from './cleanupSemantic';
  >  8 | import type { DiffOptions } from './types';
```

tracked at https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44714

**Comment by Arlorean** [comment]
I think this is fixed in PR #19 as mentioned on [Stack Overflow](https://stackoverflow.com/a/61785980/256627).
The TypeScript version needs to be 3.8.0 or higher in package.json. (Don't forget to npm install again, as I did!)

**Comment by Hotell** [comment]
Fixed by https://github.com/microsoft/create-react-app-uifabric/pull/24
